tests/integration/test_bayesian_optimization.py

```diff
@@ -104,54 +99,54 @@ def OPTIMIZER_PARAMS() -> Tuple[
     return (
         "num_steps, acquisition_rule",
         [
-            (20, EfficientGlobalOptimization()),
-            (25, EfficientGlobalOptimization(AugmentedExpectedImprovement().using(OBJECTIVE))),
-            (
-                22,
-                EfficientGlobalOptimization(
-                    MinValueEntropySearch(  # type: ignore[arg-type]
-                        BRANIN_SEARCH_SPACE,
-                        min_value_sampler=ThompsonSamplerFromTrajectory(sample_min_value=True),
-                    ).using(OBJECTIVE)
-                ),
-            ),
-            (
-                12,
-                EfficientGlobalOptimization(
-                    BatchMonteCarloExpectedImprovement(sample_size=500).using(OBJECTIVE),
-                    num_query_points=3,
-                ),
-            ),
-            (12, AsynchronousOptimization(num_query_points=3)),
-            (
-                10,
-                EfficientGlobalOptimization(
-                    LocalPenalization(
-                        BRANIN_SEARCH_SPACE,
-                    ).using(OBJECTIVE),
-                    num_query_points=3,
-                ),
-            ),
-            (
-                10,
-                AsynchronousGreedy(
-                    LocalPenalization(
-                        BRANIN_SEARCH_SPACE,
-                    ).using(OBJECTIVE),
-                    num_query_points=3,
-                ),
-            ),
-            (
-                10,
-                EfficientGlobalOptimization(
-                    GIBBON(  # type: ignore[arg-type]
-                        BRANIN_SEARCH_SPACE,
-                    ).using(OBJECTIVE),
-                    num_query_points=2,
-                ),
-            ),
+            # (20, EfficientGlobalOptimization()),
```

Comment:
Uncomment.

Also, I assume the AugmentedExpectedImprovement test wasn't always passing so you upped the iterations? Either way, remember to manually kick off an deploy-checks run before merge.

Reply:
eh, didnt save the file so these comments were a leftover - yes, had to up the iterations